### ISSUE TITLE
Switching shoulder has no effect

### ISSUE THREAD

**Issue by krojew** [Sixze/ALS-Refactored#567]
**Describe the bug**
When switching shoulder, there's no effect on camera position.

**To reproduce**
1. Open the playground map.
2. Press "C" to switch shoulder.
3. Observe no change.

**Expected behavior**
Camera moves to the other shoulder.

**Additional context**
The problem exists both in the latest release and the main branch.

**Comment by EnterTG** [comment]
I have the version from Thu Sep 12 17:35:55 2024 and do not have this problem. Try to check on a clean project.

**Comment by krojew** [comment]
Turns out changing camera is incompatible with different VisibilityBasedAnimTickOption settings.

**Comment by Sixze** [comment]
Can't reproduce. Please provide more details.

**Comment by krojew** [comment]
The easiest way to reproduce the issue is to add:

```
[/Script/Engine.SkeletalMeshComponent]
VisibilityBasedAnimTickOption=OnlyTickPoseWhenRendered
```

to DefaultEngine.ini.

**Comment by Sixze** [comment]
Fixed in d5adf6922039f35a8ef0e54b7da5290c91547a15, but you should avoid changing this setting via the config file and instead change it individually on specific components.
